components/toast/Toast.js

```diff
@@ -74,18 +109,9 @@ class Toast extends PureComponent {
                 {label}
                 {children}
               </TextBody>
-              {onClose ? (
-                action ? (
-                  <LinkButton className={theme['action-link']} inverse label={action} onClick={onClose} />
-                ) : (
-                  <IconButton
-                    className={theme['action-button']}
-                    icon={<IconCloseMediumOutline />}
-                    color="white"
-                    onClick={onClose}
-                  />
-                )
-              ) : null}
+              {this.renderCustomAction(action, actionLabel) ||
+                this.renderCustomLink(link, linkLabel, linkTarget) ||
+                this.renderCloseButton(onClose)}
```

Comment:
PP: As you're not doing anything else with these values besides passing them through, I would get the values out of props in the respective render methods:

```jsx
renderCustomAction = () => {
	const { action, actionLabel } = this.props;
    return action && <LinkButton className={theme['action-link']} inverse label={actionLabel} onClick={action} />;
}

render () {
	return (
		<div>
			{
				this.renderCustomAction() || 
				this.renderCustomLink() || 
				this.renderCloseButton()
			}
		</div>
	);
}
```